packages/core/components/AnnotationFilterForm/index.tsx

```diff
@@ -33,6 +33,7 @@ export default function AnnotationFilterForm(props: AnnotationFilterFormProps) {
     const dispatch = useDispatch();
     const allFilters = useSelector(selection.selectors.getFileFilters);
     const fuzzyFilters = useSelector(selection.selectors.getFuzzyFilters);
+    const isQueryingSourceWithFuzzy = useSelector(selection.selectors.isQueryingAicsFms);
```

Comment:
This name feels like what `fuzzySearchEnabled` is trying to describe. I think this is meant to describe whether or not we _can_ query the data with `fuzzy` so IMO I think something like `canFuzzySearch` would be clearer